node/test/client/src/block_builder.rs

```diff
@@ -67,6 +64,29 @@ impl InitPolkadotBlockBuilder for Client {
 
 		let timestamp = last_timestamp + minimum_period;
 
+		// `SlotDuration` is a storage parameter type that requires externalities to access the value.
+		let slot_duration = BasicExternalities::new_empty()
+			.execute_with(|| polkadot_test_runtime::SlotDuration::get());
+
+		let slot = (timestamp / slot_duration).into();
+
+		let digest = Digest {
+			logs: vec![
+				DigestItem::PreRuntime(
+					BABE_ENGINE_ID,
+					PreDigest::SecondaryPlain(SecondaryPlainPreDigest {
+						slot,
+						authority_index: 42,
+					}).encode()
+				),
+			],
+		};
```

Comment:
@andresilva I had to add this digest so that the test `grandpa_pause_voting_rule_works` succeed.
I'm not sure it is ok, or if it better to actually remove babe from OnTimestampSet directly.
But seems fine.